orttraining/orttraining/training_ops/cuda/collective/nccl_kernels.cc

```diff
@@ -30,6 +31,15 @@ Status NcclAllReduce::ComputeInternal(OpKernelContext* context) const {
 }
 
 NcclAllGather::NcclAllGather(const OpKernelInfo& info) : NcclKernel(info) {
+  info.GetAttrOrDefault<int64_t>("max_group_size", &max_group_size_, static_cast<int64_t>(0));
+  info.GetAttrOrDefault<int64_t>("num_input_readies", &num_input_readies_, static_cast<int64_t>(0));
+  if (max_group_size_ > 0) {
+    ORT_ENFORCE(info.GetAttrs<int64_t>("partition", partition_).IsOK());
```

Comment:
consider ORT_THROW_IF_ERROR() for expressions that return Status to get the status message too

Reply:
Yes... but this is a constructor, cannot return status.